examples/private/main.tf

```diff
@@ -0,0 +1,224 @@
+resource "aws_security_group" "client_sec_group" {
+  name        = "${var.deployment_prefix}-client-sg"
+  description = "client security group"
+  vpc_id      = var.vpc_id
+  tags        = var.tags
+
+  ingress {
+    description = "Allow outbound to ssh"
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
I'd like to see us start controlling inbound/outbound access a little better. can we make ssh inbound access a variable that defaults to 0.0.0.0/0, rather than hard coding?

Reply:
This is an example tho, not part of the module. it's also not hard code?

Reply:
fair enough.